src/main/java/de/themorpheus/edu/taskservice/endpoint/dto/response/solution/GetFreestyleSolutionResponseDTO.java

```diff
@@ -0,0 +1,21 @@
+package de.themorpheus.edu.taskservice.endpoint.dto.response.solution;
```

Comment:
Delete Class bc unneccessary

Reply:
Why? I need it to return my solution.

Reply:
If you just need it to return getFreestyleSolution delete it 
else don't